### ISSUE TITLE
Parameter hints should not use aria.alert

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#90874]
Instead they should use `activeDescendent` same as suggest widget.
Code pointer where parameterHints are currently using `aria alert` https://github.com/microsoft/vscode/blob/8451f8bd913914aac17b6b3273b6ed0db137a217/src/vs/editor/contrib/parameterHints/parameterHintsWidget.ts#L251

As @pawelurbanski points out:
![image](https://user-images.githubusercontent.com/1926584/74727867-1745f000-5242-11ea-95e8-e1197d2f15f1.png)

**Comment by isidorn** [comment]
@octref since we go on a small team event next week I will not have time to look into this.
Will you have time to look into this? It should not be too complex.
You should basically do what suggest widget does, let the editor know that the parameterHints are the `activeDescendent`.
[Code pointer as an example](https://github.com/microsoft/vscode/blob/eb36a76298b9fca0c63e62f4ebf754fd1eba6ec3/src/vs/editor/contrib/suggest/suggestWidget.ts#L801)
And just make sure to reset the activeDescendent once you are no longer visible.
To test it out just use VoiceOver / NVDA and make sure they get read out.

**Comment by pawelurbanski** [comment]
@octref ping me with any questions. At the end of the day screen reader scripts or add-ons are for fixing such corner cases just like VS extensions expand its features.

**Comment by isidorn** [comment]
I have answered @octref questsions via chat and now all should be clear. Thus removing needs more info label

**Comment by pawelurbanski** [comment]
Great timing! I’ve just pulled the changes from Master and wanted to ceate a branch for this one.

**Comment by isidorn** [comment]
Created this PR which fixes this issue https://github.com/microsoft/vscode/pull/93606
However if we merge that PR we would hit this issue again https://github.com/nvaccess/nvda/issues/10887
Thus resulting in a worse experience I believe. Due to that for now not merging this in, until that issue gets a resolution.

**Comment by isidorn** [comment]
So we would first have to tackle https://github.com/microsoft/vscode/issues/89098#issuecomment-1031505427 before we do the `activedescendant` work.
The PR I did for this issue still looks like a appropriate approach https://github.com/microsoft/vscode/pull/93606/files
